.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "standard-react"
+  ],
+  "plugins": [
+    "babel",
+    "react",
+    "promise"
```

Comment:
I think `promise` plugins is not general enough, i.e. not every project/team needs it. As a template, I think we should keep the setting minimal, so let's remove it.